pylint/pyreverse/stdlib.py

```diff
@@ -0,0 +1,359 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""
+Shim to support Python versions < 3.10 that don't have sys.stdlib_module_names.
+
+These values were created by cherry-picking the commits from
+https://bugs.python.org/issue42955 into each version, but may be updated
+manually if changes are needed.
+"""
+
+import sys
+
+# Standard library modules common to 3.7 - 3.9
```

Comment:
```suggestion
# Standard library modules common to 3.7 - 3.9
# TODO: Remove when python 3.9 isn't supported anymore 
```

Reply:
Ended up removing this file. Have a PR open to add it to Astroid.